src/components/widgets/CostOfRestWidget.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+
+import {XYPlotTemplate} from 'components/widgets/shared/XYPlotTemplate';
+import {IDefaultWidgetProps} from 'components/widgets/shared/IDefaultWidgetProps';
+import {
+  BasicHorizontalAxis,
+  BasicVerticalAxis
+} from 'components/widgets/shared/BasicAxes';
+import {LinearAreaSeries} from 'components/widgets/shared/BasicAreaSeries';
+import * as util from './shared/util';
+
+
+
+function timeCompare(a: {x: any;}, b: {x: any;}) {
+  let timeA = a.x;
+  let timeB = b.x;
+  let comparison = 0;
+
+  if (timeA > timeB) {
+    comparison = 1;
+  } else if (timeA < timeB) {
+    comparison = -1;
+  }
+  return comparison;
+}
+
+
+export interface ICostOfRestWidgetProps extends IDefaultWidgetProps {}
+
+export const CostOfRestWidget: React.FC<ICostOfRestWidgetProps> = props => (
+    <XYPlotTemplate
+    {...console.log(props)}
+    heightPx={props.heightPx}
+    widthPx={props.widthPx}
+    status={props.data.length > 2 && props.data.length > 2 ? 'ready' : 'loading'}
```

Comment:
This is probably an artifact of something I wrote, but only need one `props.data.length > 2`

Reply:
Yep fixed